.golangci.yml

```diff
@@ -62,6 +57,12 @@ linters:
   - whitespace
 
 linters-settings:
+  govet:
+    disable:
+      - printf
+  gosec:
+    excludes:
+      - G115
```

Comment:
Let's add a comment that this is e.g. " integer overflow conversion int -> int32"

Reply:
done